examples/service_envoy/listeners.pb

```diff
@@ -0,0 +1,16 @@
+listeners {
```

Comment:
I'm a little confused what this is / what format this is. This doesn't seem to match a proto definition?